xcm/xcm-builder/src/origin_conversion.rs

```diff
@@ -170,7 +178,7 @@ where
 		// We institute a root fallback so root can always represent the context. This
 		// guarantees that `successful_origin` will work.
 		if o.caller() == Origin::root().caller() {
-			Ok(MultiLocation::Here)
+			Ok(MultiLocation::here())
 		} else {
 			Err(o)
 		}
```

Comment:
everything in this file is correct, but syntactically I think we can simplify it a bit and remove some of the conditions.